cmd/api/src/api/v2/cypher_search.go

```diff
@@ -34,27 +36,68 @@ type CypherSearch struct {
 
 func (s Resources) CypherSearch(response http.ResponseWriter, request *http.Request) {
```

Comment:
Given the significant extra logic for mutation Cypher operations, I wonder if it would be worth a refactor to have two separate functions.